src/Config.php

```diff
@@ -119,7 +119,7 @@ private function setConsumerGroup(
         int $partition = null,
         int $offset = null
     ): void {
-        if (!$consumerGroupId && count($topicConfig['consumer-groups']) === 1) {
+        if (!$consumerGroupId && 1 === count($topicConfig['consumer-groups'])) {
```

Comment:
I don't see much value in this change. yoda comparison is very confusing  and hard to reason about it.
tests should do the trick in these scenarios

Reply:
I guess that the idea here is to use our defined standard on all our repos. If you specifically disagree with the standard, we should discuss and change it, not create exceptions for our taste on specific repos. Otherwise it would not be a standard.

Reply:
should we maintain as it is now (StyleCi, because its working properly), and discuss this another time?

Reply:
What do you think are the advantages of this approach ? Do you think that the `phpcs` does not work properly?

Reply:
ping @brnbp

Reply:
you can merge, i've nothing to say anymore.